js/MainContent.js

```diff
@@ -270,12 +270,13 @@ function validateUNHCR($elem, throwErrorFlag) {
             msg = message;
         } else {
             msg =
-                "UNHCR numbers must match one of these formats:" +
-                "\\n###-YYC##### (new)" +
-                "\\n####/YYYY (old)" +
-                "\\n###-AP######## (new appointment slip)" +
-                "\\n###-CS######## (old appointment slip)" +
-                '\\nFor no UNHCR number, enter \\"None\\"';
+                `UNHCR numbers must match one of these formats:
+                \\n###-YY-##### (new)
+                \\n###-YYC##### (old)
+                \\n####/YYYY (very old)
+                \\n###-AP######## (new appointment slip)
+                \\n###-CS######## (old appointment slip)
+                \\nFor no UNHCR number, enter "None"`;
```

Comment:
Hmm In my opinion, it is a bit better to keep the old way here, so that when we change 1 line (a new format), it's easy to see the diff here is just one little line. Also, using back-ticks keeps whitespace so this can have unintended side-effects. Did you tests this in RIPS? If yes, can you take a screenshot and show it in the PR?

Reply:
![image](https://user-images.githubusercontent.com/42237442/133622336-46a0d08c-5a3f-4f88-a1f8-1fc9fdeb49a0.png)

This is how it looks in the RIPS. do you still want it to be the old style?

Reply:
> Hmm In my opinion, it is a bit better to keep the old way here, so that when we change 1 line (a new format), it's easy to see the diff here is just one little line. Also, using back-ticks keeps whitespace so this can have unintended side-effects. Did you tests this in RIPS? If yes, can you take a screenshot and show it in the PR?


![image](https://user-images.githubusercontent.com/42237442/133622336-46a0d08c-5a3f-4f88-a1f8-1fc9fdeb49a0.png)

This is how it looks in the RIPS. do you still want it to be the old style?

Reply:
That's nice that it ends up displaying well :D We can change it later if we want, it's fine for now haha